apps/divorce/div-cfs/div-cfs.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       startupPeriod: 120
       startupFailureThreshold: 3
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/div/cfs:prod-069d789-20230117100654 #{"$imagepolicy": "flux-system:div-cfs"}
+      image: hmctspublic.azurecr.io/div/cfs:pr-293-5bb3ae4-20230118182847 #{"$imagepolicy": "flux-system:demo-div-cfs"}
```

Comment:
ALso this one, create a demo.yaml rather than edit the base yaml file

Reply:
Done